src/riak_repl2_rtsource_conn.erl

```diff
@@ -463,7 +463,7 @@ send_heartbeat(State = #state{hb_timeout = HBTimeout,
 
     % Using now as need a unique reference for this heartbeat
     % to spot late heartbeat timeout messages
-    Now = now(),
+    Now = os:timestamp(),
```

Comment:
The comment (two lines above) says it’s `now()` and it’s unique.

Reply:
If you look at the code, this is only used for estimating RTT for statistics.  I think it's fine, and it doesn't appear to require it to be monotonic nor globally unique.